test/test_key.cc

```diff
@@ -62,6 +64,10 @@ int main(int, char**) {
     client.KeyRm("foobar-key");
     /** [ipfs::Client::KeyRm] */
 
+    client.KeyRename("foobar-key2", "foobar-new-key2");
```

Comment:
Surround your test code with:  `/** [ipfs::Client::KeyRename] */` in order to have the doxygen snipped feature working.